gapis/replay/asm/instructions.go

```diff
@@ -390,3 +390,14 @@ func (a Notification) Encode(r value.PointerResolver, w binary.Writer) error {
 	}
 	return opcode.Notification{}.Encode(w)
 }
+
+type Wait struct {
+	ID uint64
+}
+
+func (a Wait) Encode(r value.PointerResolver, w binary.Writer) error {
+	if err := encodePush(protocol.Type_Uint64, a.ID, w); err != nil {
```

Comment:
Instead of push-popping the id, can we make it an immediate to the instruction? I don't think we'll need that many IDs.

Reply:
I'll pack it into the instruction, but the existing convention seems to be to unpack it in interpreter.cpp and push it onto the stack anyways.